packages/cli/register/package.json

```diff
@@ -37,6 +37,7 @@
     "@fern-api/fs-utils": "workspace:*",
     "@fern-api/ir-generator": "workspace:*",
     "@fern-api/ir-sdk": "workspace:*",
+    "@fern-api/ir-utils": "workspace:*",
```

Comment:
Nice - I could imagine a ton of the IR data going in here once we're in an all TypeScript world (e.g. all the casing conventions). This is probably what you had in mind?

Reply:
Yup, all the casing conventions are already there! will slowly move everything not fern def specific there and share